src/Materials/Textures/babylon.videoTexture.ts

```diff
@@ -92,18 +92,21 @@
             if (typeof src === "object") {
                 return src.toString();
             }
-    
+
             return src;
         };
     
         private _getVideo(src: string | string[] | HTMLVideoElement): HTMLVideoElement {
             if (src instanceof HTMLVideoElement) {
+                Tools.SetCorsBehavior(src.currentSrc, src);
                 return src;
             }
             const video: HTMLVideoElement = document.createElement("video");
             if (typeof src === "string") {
+				Tools.SetCorsBehavior(src, video);

```

Comment:
tabbing seems off

Reply:
sorry about that, just updated webstorm and apparently it decided this document full of spaces should use tabs for it's tabbing on those lines. Used to behave better than that, so I forgot to check. I'll fix this a quick second.